src/components/vault/vault.jsx

```diff
@@ -666,6 +677,29 @@ class Vault extends Component {
     )
   }
 
+  renderStrategyRewards = () => {
+
+    const { classes } = this.props
+
+    return( 
+          <div className={ classes.disaclaimer } style={{ marginTop: '25px', maxWidth: '500px' }}>
+
+            <Grid container spacing={1}>
+              <Grid item><TimelineIcon fontsize="small" /></Grid>
+              <Grid item xs>
+                <Typography variant="h4" style={{ display: 'inline', fontWeight: 'bold' }}>
+                Strategy Rewards
+                </Typography>
+              </Grid>
+            </Grid>
+
+            <Typography variant="body1">
+              Yearn is only possible due to community contributions. Vault strategy contributors are rewarded with <b>0.5%</b> of yield generated from by the vault. <a href="https://docs.yearn.finance/how-to-guides/how-to-contribute-a-yvault-strategy/" target="_blank">Learn more &rarr;</a>
```

Comment:
Where did we land on this @wirjo ? Still pending change?

Reply:
I think this is fine now but maybe @milkyklim should just make a call and merge this.

Reply:
Please, merge if code it fine @x48-crypto.